src/game.rs

```diff
@@ -54,18 +50,21 @@ impl Game {
 
     pub fn run(&mut self, window: &mut PistonWindow, opengl: &mut GlGraphics, window_size: &Size) {
         while let Some(event) = window.next() {
-            Self::wrap_position(&mut self.position, window_size);
-
+            //Self::wrap_position(&mut self.position, window_size);
+            self.player.setWindowSize(window_size.width, window_size.height);
```

Comment:
Does `setWindowSize` need to be called every time in the `while` loop or could it just be set once before the loop starts?